app/Http/Controllers/PostsController.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Comment;
+use App\Models\Posts;
+use Illuminate\Http\Request;
+
+class PostsController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+
+    public function index()
+    {
+        $posts = Posts::latest()->get();
+
+        return view('posts.index', compact('posts'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('posts.create');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+
+        $this->validate($request, ['title' => 'required', 'body' => 'required']);
+
+        Posts::create([
+            'title' => $request->title,
+            'body' => $request->body,
+            'user_id' => auth()->id(),
+            'published_at' => now(),
+            'updated_at' => now(),
+            'created_at' => now(),
+        ]);
+
+        return redirect('posts');
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Models\Posts  $posts
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $post = Posts::findOrFail($id);
+        $comments = Comment::where('post_id', $id)->get();
+
+        return view('posts.show', ['comments' => $comments, 'post' => $post]);
```

Comment:
ne ti trqbva da vzema6 komentarite. Prosto vyv view-to 6te ima6: `$post->comments`, 4iqto vryzka trqbva da ima6 v modela (vse o6te ne sym go gledal). T.e. izpra6tai samo posta bez komentarite kym view-to.